src/app/shared/entry.ts

```diff
@@ -124,6 +125,13 @@ export abstract class Entry implements OnDestroy {
     }
   }
 
+  protected isAppTool(url: String): boolean {
+    if (url.includes('containers') && url.includes('github.com')) {
```

Comment:
If I have a workflow with `containers` in its name, will this return `true`?

Reply:
yes, it will return true

Reply:
Wouldn't that be wrong? It's a real edge case (I don't see any workflows with "containers" in their giturl or workflow name in the current DB), but it doesn't seem like an unreasonable word to use either. I don't think it would be that hard to catch the case -- not sure what the exact url is, but either a regex or `url.includes('/containers/')` or something like that.

Reply:
Yeah, you're right. It would be wrong. I'll change the regex to `url.includes('/containers/github.com')`